[2025-01-30T10:26:00.001+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T10:26:00.026+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2020-05-02T00:00:00+00:00 [queued]>
[2025-01-30T10:26:00.052+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2020-05-02T00:00:00+00:00 [queued]>
[2025-01-30T10:26:00.053+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-30T10:26:00.089+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upload_to_cloud> on 2020-05-02 00:00:00+00:00
[2025-01-30T10:26:00.101+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=384) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-30T10:26:00.102+0000] {standard_task_runner.py:72} INFO - Started process 388 to run task
[2025-01-30T10:26:00.108+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fhv_to_gcp_dag', 'upload_to_cloud', 'scheduled__2020-05-02T00:00:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/homework_fhv_dag.py', '--cfg-path', '/tmp/tmp3gp0uagl']
[2025-01-30T10:26:00.116+0000] {standard_task_runner.py:105} INFO - Job 88: Subtask upload_to_cloud
[2025-01-30T10:26:00.219+0000] {task_command.py:467} INFO - Running <TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2020-05-02T00:00:00+00:00 [running]> on host 9573d401388f
[2025-01-30T10:26:00.359+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fhv_to_gcp_dag' AIRFLOW_CTX_TASK_ID='upload_to_cloud' AIRFLOW_CTX_EXECUTION_DATE='2020-05-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2020-05-02T00:00:00+00:00'
[2025-01-30T10:26:00.363+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-30T10:26:02.208+0000] {logging_mixin.py:190} INFO - The file /opt/***/data/fhv_2020-05_data.parquet has been uploaded to nrabbi-33-dtc-de-data-bucket/<Blob: nrabbi-33-dtc-de-data-bucket, /data/fhv_2020-05_data.parquet, 1738232762235864>
[2025-01-30T10:26:02.208+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-30T10:26:02.222+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T10:26:02.223+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fhv_to_gcp_dag, task_id=upload_to_cloud, run_id=scheduled__2020-05-02T00:00:00+00:00, execution_date=20200502T000000, start_date=20250130T102600, end_date=20250130T102602
[2025-01-30T10:26:02.264+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-30T10:26:02.286+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-30T10:26:02.287+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
